.circleci/config.yml

```diff
@@ -27,15 +29,31 @@ workflows:
             tags:
               only: /^v.*/
 
+      # build and push docker image to docker.
```

Comment:
Added because of https://gigantic.slack.com/archives/C04TGHDEF/p1630672178006900